### PR TITLE
Add stub template for App Runtime Platform WG

### PR DESCRIPTION
**Pull request by emalm**
This PR proposes a charter document for the App Runtime Platform working group.

### CODE REVIEW

**Comment by Gerg** [comment]
I would love to see https://github.com/cloudfoundry-incubator/haproxy-boshrelease promoted to the `cloudfoundry` organization.

I believe it is the only "Technical Asset" in the list that is still under incubation. At ~600 commits and numerous production deployments, it seems ready to hatch 🐣.

**Comment by emalm** [comment]
@Gerg SGTM! The TOC has been discussing merging the entire cloudfoundry-incubator org into the cloudfoundry org, likely after a long-overdue round of repo pruning/archival.

**Comment by emalm** [comment]
FYI @aminjam @Benjamintf1 @ameowlia @gcapizzi @geofffranks @domdom82 for feedback and suggestions for approvers/leads.

**Comment by domdom82** [comment]
@emalm I agree. I think we have a list of things to do besides clicking the "transfer repository" button. I volunteer for taking ownership since @geoffranks phased out of the project quite some time ago. I might need to have the rights to create a repo under the cloudfoundry org, though.

**Comment by emalm** [comment]
@ameowlia I added some initial goals and scope to this WG's draft charter and proposed you as a lead in the group. Could you please provide feedback on them and work with the TOC to establish a good initial charter for this WG? Thanks!

File: toc/working-groups/app-runtime-platform.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## Proposed Membership
+
+- Technical Lead(s): @ameowlia, others TBD
+- Execution Lead(s): TBD
+- Approvers: TBD
+
+
+## Technical Assets
+
+Components from the Diego, Eirini, Garden, HAproxy, Logging and Metrics, Networking, Windows Containers projects.
```
Comment (ameowlia) [inline toc/working-groups/app-runtime-platform.md:41]:
```suggestion
Components from the Diego, Eirini, Garden, HAproxy, Logging and Metrics, Networking, Windows Containers projects.

### Diego
* https://github.com/cloudfoundry/diego-release
  * https://github.com/cloudfoundry/archiver
  *  https://github.com/cloudfoundry/auction
  *  https://github.com/cloudfoundry/auctioneer
  *  https://github.com/cloudfoundry/bbs
  *  https://github.com/cloudfoundry/benchmarkbbs
  *  https://github.com/cloudfoundry/bytefmt
  *  https://github.com/cloudfoundry/cacheddownloader
  *  https://github.com/cloudfoundry/certsplitter
  *  https://github.com/cloudfoundry/cfdot
  *  https://github.com/cloudfoundry/clock
  *  https://github.com/cloudfoundry/consuladapter
  *  https://github.com/cloudfoundry/debugserver
  *  https://github.com/cloudfoundry/diego-dockerfiles
  *  https://github.com/cloudfoundry/diego-logging-client
  *  https://github.com/cloudfoundry/diego-upgrade-stability-tests
  *  https://github.com/cloudfoundry/dockerdriver
  *  https://github.com/cloudfoundry/durationjson
  *  https://github.com/cloudfoundry/ecrhelper
  *  https://github.com/cloudfoundry/eventhub
  *  https://github.com/cloudfoundry/executor
  *  https://github.com/cloudfoundry/fileserver
  *  https://github.com/cloudfoundry/healthcheck
  *  https://github.com/cloudfoundry/inigo
  *  https://github.com/cloudfoundry/localdriver
  *  https://github.com/cloudfoundry/localip
  *  https://github.com/cloudfoundry/locket
  *  https://github.com/cloudfoundry/operationq
  *  https://github.com/cloudfoundry/rep
  *  https://github.com/cloudfoundry/route-emitter
  *  https://github.com/cloudfoundry/systemcerts
  *  https://github.com/cloudfoundry/tlsconfig
  *  https://github.com/cloudfoundry/vizzini
  *  https://github.com/cloudfoundry/volman
  *  https://github.com/cloudfoundry/workpool

### Eirini
* https://github.com/cloudfoundry-incubator/eirini

### Garden
* https://github.com/cloudfoundry/garden-runc-release
  * https://github.com/cloudfoundry/dontpanic/
  * https://github.com/cloudfoundry/garden
  * https://github.com/cloudfoundry/garden-integration-tests
  * https://github.com/cloudfoundry/grootfs
  * https://github.com/cloudfoundry/guardian
  * https://github.com/cloudfoundry/idmapper
  * https://github.com/cloudfoundry/netplugin-shim

### HAproxy
* https://github.com/cloudfoundry-incubator/haproxy-boshrelease

### Logging and Metrics
* https://github.com/cloudfoundry/bosh-system-metrics-forwarder-release
* https://github.com/cloudfoundry/log-cache-release
* https://github.com/cloudfoundry/loggregator-release
* https://github.com/cloudfoundry/loggregator-agent-release
* https://github.com/pivotal-cf/metric-registrar-release
* https://github.com/pivotal-cf/metric-registrar-cli
* https://github.com/cloudfoundry/metrics-discovery-release
* https://github.com/cloudfoundry/statsd-injector-release
* https://github.com/cloudfoundry/syslog-release
* https://github.com/cloudfoundry/system-metrics-scraper-release

### Networking
* http://github.com/cloudfoundry/routing-release
  * https://github.com/cloudfoundry/routing-acceptance-tests
  * https://github.com/cloudfoundry/routing-api
  * https://github.com/cloudfoundry/routing-api-cli
  * https://github.com/cloudfoundry/routing-perf-release
  * https://github.com/cloudfoundry/cf-tcp-router
  * https://github.com/cloudfoundry/routing-info
  * https://github.com/cloudfoundry/gorouter
  * https://github.com/cloudfoundry/route-registrar
* https://github.com/cloudfoundry/cf-networking-release
  * https://github.com/cloudfoundry/networking-oss-deployments
  * https://github.com/cloudfoundry/cf-networking-helpers
* https://github.com/cloudfoundry/silk-release
  * https://github.com/cloudfoundry/silk/
* https://github.com/cloudfoundry/nats-release


### Windows Containers
* https://github.com/cloudfoundry/winc-release
  * https://github.com/cloudfoundry/winc
  * https://github.com/cloudfoundry/groot-windows
  * https://github.com/cloudfoundry-incubator/diff-exporter
  * https://github.com/cloudfoundry/cert-injector
```

**Comment by ameowlia** [comment]
I'm not sure how many approvers we are expecting in each area. I'm happy to take a "more is more" approach and start with high trust.

Suggested Approvers (if not Tech Leads 👀 ):
* @domdom82 
* @46bit
* @stefanlay
* @aminjam
* @Benjamintf1 
* @gcapizzi
* @geofffranks
* @selzoc
* @Gerg 
* @pivotalgeorge
* @mkocher 
* @acrmp 
* @jrussett
* @reneighbor

File: toc/working-groups/app-runtime-platform.md [hunk 2]
```diff
@@ -26,14 +26,19 @@ Provides operational components for the CF App Runtime, including those for appl
...
 ## Proposed Membership
 
 - Technical Lead(s): @ameowlia, others TBD
 - Execution Lead(s): TBD
-- Approvers: TBD
+- Approvers by area:
+  - Diego: @acrmp, @aminjam, @Benjamintf1, @jrussett, @pivotalgeorge, @reneighbor, @selzoc
+  - Eirini: @danail-branekov, @gcapizzi, @georgethebeatle, @kieron-dev, @mnitchev
+  - Garden Containers: @aemengo, @danail-branekov, @gcapizzi, @georgethebeatle, @kieron-dev, @mnitchev
+  - Logging and Metrics: @acrmp, @aminjam, @Benjamintf1, @jrussett, @pivotalgeorge, @reneighbor, @selzoc
```
Comment (ameowlia) [inline toc/working-groups/app-runtime-platform.md:40]: Can we add @geofffranks to Diego and Logging+Metics,

**Comment by ameowlia** [comment]
I'd also like to nominate Ivan Hristov ( @IvanHristov98 ) from SAP as an approver for the Diego repos.

File: toc/working-groups/app-runtime-platform.md (hunk 2 above)
Comment (emalm) [reply toc/working-groups/app-runtime-platform.md]: Done!

**Comment by emalm** [comment]
@ameowlia I've updated the PR draft with those additional approver candidates. Please let me know whether you think the charter draft is ready for the TOC to review officially or whether there should be any more changes before that!

File: toc/working-groups/app-runtime-platform.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+- @acrmp
+- @aminjam
+- @Benjamintf1
+- @geofffranks
+- @jrussett
+- @pivotalgeorge
+- @reneighbor
+- @selzoc
+
+Networking:
```
Comment (ameowlia) [inline toc/working-groups/app-runtime-platform.md:74]: I'm nominating Patrick Lowin (@plowin) from SAP to be an approver in the networking area. He is a current maintainer of the haproxy bosh release.

Comment (emalm) [reply toc/working-groups/app-runtime-platform.md:74]: Done!

**Review by stephanme — APPROVED** [review]
LGTM

**Review by loewenstein — COMMENTED** [review]
Just noticed that there is no Eirini mentioned in the approver or technical assets groups. Kubernetes is still mentioned in the goals of though.

**Comment by emalm** [comment]
@loewenstein Good point, I was waffling on whether to include K8s infrastructure in the goals as a longer-term view, but I think it's better to remove it for now. Then there's no update needed if the new CF on K8s WG ends up being long-lived, and if we do end up merging areas from it into the other App Runtime WGs then that should prompt a change to goals and scope anyway.

**Review by loewenstein — APPROVED** [review]

**Review by LeePorte — APPROVED** [review]

**Pull request merged**